cloudigrade/account/tests/test_reports_aws.py

```diff
@@ -342,3 +343,356 @@ def test_usage_on_times_overlapping(self):
+             'name': self.account.name,
+             'images': 2,
+             'instances': 2,
+             'rhel_instances': 1,
+             'openshift_instances': 1}
+        self.assertEqual(expected_overview, overview)
+
+    def test_get_cloud_account_overview_with_two_instances_same_image(self):
+        """Assert an account overview with events reports correctly."""
+        # generate event for instance_1 with the rhel/openshift image
+        account_helper.generate_single_aws_instance_event(
+            self.instance_1, self.start, InstanceEvent.TYPE.power_on,
+            self.openshift_and_rhel_image.ec2_ami_id)
+        # generate event for instance_2 with the rhel/openshift image
+        account_helper.generate_single_aws_instance_event(
+            self.instance_2, self.start, InstanceEvent.TYPE.power_on,
+            self.openshift_and_rhel_image.ec2_ami_id)
+        overview = reports.get_account_overview(
+            self.account, self.start, self.end)
+        # assert that we only find the one image
+        expected_overview = \
+            {'id': self.account.aws_account_id,
+             'user_id': self.account.user_id,
+             'type': AWS_PROVIDER_STRING,
+             'arn': self.account.account_arn,
+             'creation_date': self.account.created_at,
+             'name': self.account.name,
+             'images': 1,
+             'instances': 2,
+             'rhel_instances': 1,
+             'openshift_instances': 1}
+        self.assertEqual(expected_overview, overview)
+
+    def test_get_cloud_account_overview_with_rhel(self):
+        """Assert an account overview with events reports correctly."""
+        # generate event for instance_1 with the rhel/openshift image
+        account_helper.generate_single_aws_instance_event(
+            self.instance_1, self.start, InstanceEvent.TYPE.power_on,
+            self.openshift_and_rhel_image.ec2_ami_id)
+        # generate event for instance_2 with the rhel image
+        account_helper.generate_single_aws_instance_event(
+            self.instance_2, self.start, InstanceEvent.TYPE.power_on,
+            self.rhel_image.ec2_ami_id)
+        overview = reports.get_account_overview(
+            self.account, self.start, self.end)
+        # assert that we only find the two rhel images
+        expected_overview = \
+            {'id': self.account.aws_account_id,
+             'user_id': self.account.user_id,
+             'type': AWS_PROVIDER_STRING,
+             'arn': self.account.account_arn,
+             'creation_date': self.account.created_at,
+             'name': self.account.name,
+             'images': 2,
+             'instances': 2,
+             'rhel_instances': 2,
+             'openshift_instances': 1}
+        self.assertEqual(expected_overview, overview)
+
+    def test_get_cloud_account_overview_with_openshift(self):
+        """Assert an account overview with events reports correctly."""
```

Comment:
This docstring is identical to the ones for several preceding test functions. They should have enough information to distinguish different tests.